src/components/Product/ProductItem.jsx

```diff
@@ -70,14 +83,34 @@ const ProductItem = ({ product }) => {
             />
           </Grid>
           <Grid item lg={2} className={classes.centerItem}>
-            <CheckCircleOutlineIcon className={classes.circleIcon} />
+          {product.allergens ? (
+            <Paper className={classes.paperWarning}>
+              <WarningRoundedIcon className={classes.warningRoundedIcon} />
+              {/* <Typography> Contains </Typography> */}
+            </Paper>
+          ) : (
+            <Paper className={classes.paperCheck}>
+              <CheckCircleOutlineIcon className={classes.checkIcon} />
```

Comment:
I used CheckCircleRoundedIcon on product detail If you can change it please, Its more rounded and greener